apps/dashboard/src/components/conditions-editor/value-editor.tsx

```diff
@@ -1,13 +1,18 @@
 import React, { useMemo } from 'react';
 import { autocompletion } from '@codemirror/autocomplete';
 import { useValueEditor, ValueEditorProps } from 'react-querybuilder';
+import { useFormContext } from 'react-hook-form';
 
 import { completions } from '@/utils/liquid-autocomplete';
 import { InputRoot, InputWrapper } from '@/components/primitives/input';
 import { Editor } from '@/components/primitives/editor';
-import { variables } from '@/components/conditions-editor/conditions-editor';
+import { LiquidVariable } from '@/utils/parseStepVariablesToLiquidVariables';
 
 export const ValueEditor = React.memo((props: ValueEditorProps) => {
+  const form = useFormContext();
+  const queryPath = 'query.rules.' + props.path.join('.rules.') + '.value';
+  const { error } = form.getFieldState(queryPath, form.formState);
```

Comment:
we don't want to register fields but want just to get updates for the field errors, this is the way you do that